### PR TITLE
DevKit updates

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
some more clean ups.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 Contributing
 ------------
 
-Symfony CMF is an open source, community-driven project. We follow the same
-guidelines as core Symfony. If you'd like to contribute, please read the
+Symfony2 CMF is an open source, community-driven project. We follow the same
+guidelines as core Symfony2. If you'd like to contribute, please read the
```
Comment (dbu) [inline CONTRIBUTING.md]: there is some regression here

**Comment by ElectricMaxxx** [comment]
Indeed that seems to be a problem, i will look if dev-kit uses a fresh master always or an old one.

**Comment by ElectricMaxxx** [comment]
I think it was me when fetching the new master-dev-kit branch from next try. I had the old one. Shall we subfix them with a kind of a hash which should be stable for the current run? to force me to fetch a new branch every time?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ElectricMaxxx) [reply CONTRIBUTING.md]: that is strange

**Comment by ElectricMaxxx** [comment]
Travis is green, but does not show it here. :-(

**Comment by dbu** [comment]
please update the php version in composer.json as well, so it matches what we do on travis-ci

**Pull request merged**